excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/CaseTypeParser.java

```diff
@@ -58,6 +71,10 @@ public CaseTypeParser(ParseContext parseContext,
     }
 
     public ParseResult<CaseTypeEntity> parseAll(Map<String, DefinitionSheet> definitionSheets) {
+        return parseAll(definitionSheets, Sets.newHashSet());
+    }
+
+        public ParseResult<CaseTypeEntity> parseAll(Map<String, DefinitionSheet> definitionSheets,Set<String> missingUserRoles) {
```

Comment:
do we need to create an overloaded method? I think it would be simpler if the original method returned missingUserRoles inside the ParseResult

Reply:
I agree... with the below comment and I am going to change this